docs/radicle.md

```diff
@@ -28,39 +25,50 @@ while offering global decentralized repository storage through the peer-to-peer
 intermediaries such as hosted servers.
 
-For more on Radicle Link, check out the [Understanding Radicle][wh] section.
+The radicle-link protocol distinguishes between two types of identities: personal and project.
+The first describes an actor in the system, while the second describes a (software) project
+on which one or more actors collaborate. In Radicle:
 
-## How is Radicle different from GitHub?
+1. peers follow other peers.
+2. peers track projects they are interested in.
+3. peers gossip about projects. This means tracking and replicating objects from
+the peers they follow, about the projects they are interested in.
 
-When coming to Radicle from a centralized code collaboration platform like
-Github or Gitlab, you might notice that Radicle priotizes different concerns. In
-contrast to the traditional Web2.0 model of centralized code collaboration
-platforms (i.e. global by default, reliance on service providers, convienience),
-Radicle ensures that:
-
-1. People control their identity
-2. People control their content
-3. People control their social interactions
+It's this "trusted" social graph of peers and projects that becomes the foundation for
+collaboration within Radicle.
 
-These concerns have informed the design of [Radicle Link][rl], the peer-to-peer
-protocol that powers the Radicle network. Radicle Link gossips and replicates
-data between `peers`.
+For more on Radicle Link, check out the [Understanding Radicle][wh] section.
 
-In Radicle:
+## How is Radicle different from GitHub?
 
-1. Peers follow other peers.
-2. Peers track projects they are interested in.
-3. Peers gossip about projects - this means tracking peers can replicate objects
-   from the peers they follow, about the projects they are interested in.
+When coming to Radicle from a centralized code collaboration platform like
+Github or Gitlab, you might notice that Radicle priotizes different concerns. Concretely:
 
-These interactions create a "trusted" social graph of peers and projects that
-becomes the foundation for collaboration within Radicle.
+- People being in control of their identity
+- People being in control of their content
+- People being in control of their social interactions
 
 Read more about the [Radicle collaboration model][fa].
 
+The above concerns led us to a design and implementation that differentiates itself from
+centralized forges in a few critical ways:
+
+1. The Radicle stack is open-source from beginning to end. There are no "closed" components.
+Every component of the Radicle stack is auditable, modifiable and extendable.
+2. Radicle is build *entirely* on open protocols. There are no "special servers", privileged
+users or companies in control of your collaboration.
+3. Radicle is based on a peer-to-peer architecture instead of a client server model.
+4. All artifacts within Radicle are cryptographically signed and are verified by your client,
+including social artifacts like issues, PRs and comments.
```

Comment:
This is misleading given that git objects are disseminated as-is.

Reply:
How would you rewrite @xla ? 

Something like "All data on the Radicle network is verifiable and cryptographically signed" maybe?

Reply:
My point is that we disseminate a large amount of data which is not verifiable or cryptographically signed. So I don't think we can leave this statement in, unless qualifying it, which might be more confusing than helpful as it only applies to identity data.

Reply:
yeah that's fair actually